docs/survey.md

```diff
@@ -40,128 +44,246 @@
+## If you were concerned that a big company is creating a competing offering, would you like to tell us who that is?
 
-- ( ) Strongly agree
-- ( ) Somewhat agree
-- ( ) Neither agree nor disagree
-- ( ) Somewhat disagree
-- ( ) Strongly disagree
+* Yes: [-----------]
+* Prefer not to say
 
-## Overall, I enjoy being an open source contributor
+## What is your experience of being an open source maintainer?
 
-- ( ) Strongly agree
-- ( ) Somewhat agree
-- ( ) Neither agree nor disagree
-- ( ) Somewhat disagree
-- ( ) Strongly disagree
+**Filter: Next question is only visible if the answer is 4-5**
+
+- ( ) I enjoy it a lot
+- ( ) I mostly enjoy it
+- ( ) I feel neutral about it
+- ( ) I mostly don't enjoy it
+- ( ) I don't enjoy it all
+
+## What is the reason you don't enjoy being a open source maintainer?
+
+* [-----------]
+
+## What is your experience of being an open source contributor?
+
+**Filter: Next question is only visible if the answer is 4-5**
+
+- ( ) I enjoy it a lot
+- ( ) I mostly enjoy it
+- ( ) I feel neutral about it
+- ( ) I mostly don't enjoy it
+- ( ) I don't enjoy it all
+
+## What is the reason you don't enjoy being an open source contributor?
+
+* [-----------]
+
+## What is your experience of being an open source consumer?
+
+**Filter: Next question is only visible if the answer is 4-5**
+
+- ( ) I enjoy it a lot
+- ( ) I mostly enjoy it
+- ( ) I feel neutral about it
+- ( ) I mostly don't enjoy it
+- ( ) I don't enjoy it all
+
+## What is the reason you don't enjoy being an open source consumer?
+
+* [-----------]
 
 ## As an open source maintainer...
 
-### ...I am doing my work
+### ...where do you do most of your work?
 
 - ( ) Mostly in my spare time
 - ( ) A combination of spare time and work time
-- ( ) Mostly as part of my job (employee)
-- ( ) Mostly as part of my job (self-employed)
+- ( ) Mostly during my work time
 
-### ...I feel I can sustain the project for a long time
+### ...how difficult is it for you to maintain your project over a long time?
 
-- ( ) Strongly agree
-- ( ) Somewhat agree
-- ( ) Neither agree nor disagree
-- ( ) Somewhat disagree
-- ( ) Strongly disagree
+- ( ) Very difficult
+- ( ) Somewhat difficult
+- ( ) Neither difficult nor easy
+- ( ) Somewhat easy
+- ( ) Very easy
 
-### ...I feel I get sufficient help from the community
+### ...how difficult is it for you to get help from the community?
 
-- ( ) Strongly agree
-- ( ) Somewhat agree
-- ( ) Neither agree nor disagree
-- ( ) Somewhat disagree
-- ( ) Strongly disagree
+- ( ) Very difficult
+- ( ) Somewhat difficult
+- ( ) Neither difficult nor easy
+- ( ) Somewhat easy
+- ( ) Very easy
+
+### ...how much would money help the maintainance of your project?
+
+- ( ) It would help a lot
+- ( ) It would help somewhat
+- ( ) I'm not sure
+- ( ) It wouldn't help much
+- ( ) It wouldn't help at all
+
+### ...how easy to you feel it is to choose a license that helps you realize your open source goals?
```

Comment:
Might be good to better understand what they would use the money for?
```suggestion
#### How would money help you maintain your project?

- ( ) Pay for hosting cost (website, DNS, certificates, etc)
- ( ) Pay for cloud consumption (Azure resources to test, etc.)
- ( ) Pay for tools (GitHub Pro, Jetbrains R#, etc.)
- ( ) Get time off to the dedicated time to maintain
- ( ) Enjoy some non-work activities
- ( ) Other (explain below)

#### Explain your scenario

* [-----------]

### ...how easy to you feel it is to choose a license that helps you realize your open source goals?
```

Reply:
Good idea.